client/my-sites/plan-features/style.scss

```diff
@@ -274,6 +274,7 @@ $plan-features-sidebar-width: 272px;
 
 .popover.plan-features__header-tip-info .popover__inner {
 	white-space: nowrap;
```

Comment:
To be on the safe side, let's remove `white-space: nowrap;` and set this to 250px so that it'll display on mobile devices nicely.

![image](https://user-images.githubusercontent.com/6981253/33079772-6ebc5988-cea4-11e7-922c-6137991e8fbf.png)

![image](https://user-images.githubusercontent.com/6981253/33079839-9c64b830-cea4-11e7-96c2-48f97b801ca1.png)